src/styles.css

```diff
@@ -256,6 +260,26 @@ h3 {
   height: 97%;
 }
 
+label {
+  color: black;
+  font-family: Lexend;
+  font-size: 20px;
+  padding: 0 0 10px 30px;
+}
+
+label:hover {
+  cursor: pointer;
+  transform: scale(1.1);
+}
+
+input[type=checkbox] {
+  accent-color: rgb(0, 172, 0);
+  transform: scale(2);
+  clip-path: circle(50% at 50% 50%);
+  margin-right: 20px;
+  padding: 10px;
```

Comment:
Defining the type for an input tag in CSS doesn't provide the same semantic meaning as defining the type within the input tag in HTML. Is this a change we could make? 

If you need any further clarification on what I am referring to let me know!

Reply:
Didn't I also define it in the html? Yah might need further clarification on this one!

Reply:
Whoopsie total oversight on my part! So sorry, you DID define it in HTML. Please disregard.